packages/oui-dropdown/src/dropdown.controller.js

```diff
@@ -0,0 +1,122 @@
+import Popper from 'popper.js'
+import { addBooleanParameter } from '@oui-angular/common/component-utils'
+
+const KEY_ESCAPE = 27
+
+export default class {
+  constructor ($attrs, $document, $element, $log, $scope) {
+    'ngInject'
+
+    this.$attrs = $attrs
+    this.$document = $document
+    this.$element = $element
+    this.$log = $log
+    this.$scope = $scope
+  }
+
+  $onInit () {
+    this.isDropdownOpen = false
+    this.hasFocus = false
+    this.currentFocusedElement = null
+    addBooleanParameter(this, 'arrow')
+    addBooleanParameter(this, 'start')
+    addBooleanParameter(this, 'end')
+
+    if (this.start && this.end) {
+      this.$log.warn('A dropdown cannot be positionned at the start and at the end at the same time.')
```

Comment:
So you should use one parameter instead like dropdown-position / dropdown-alignment ?

When you declare a button, do you do :

1. button[type="submit/button/reset"]
2 button[submit][button][reset]

It's always the same question, uses always the same logic !

Reply:
There was a discussion about it this morning. Maybe it will change.

Reply:
Changed